modules/llvm/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "homepage": "https://llvm.org/",
+    "maintainers": [],
```

Comment:
We will likely want to add ourselves as maintainers in here? I'm happy to be listed.